README.md

```diff
@@ -147,13 +163,16 @@ will look like this:
   },
   "options": {}
 }
-
 ```
 
 A response from the verifier endpoint (`/credentials/verify`) for a
 verifiable credential might look like this (only HTTP response codes are
 checked):
 
+```http
+HTTP/1.1 200 OK
+```
+
 ```json
 {
   "checks": ["proof"],
```

Comment:
in addition to verified I believe it will also contain an errors etc.

This should be what the verification results look like:

https://github.com/digitalbazaar/vc/blob/9975a7b9184bb50b179566517f9d480a5d30170e/lib/index.js#L74-L88

Reply:
FWIW, it looks like that referenced code uses `presentationResult` and `credentialResults` which are not used (an longer?) in the VC API:
https://github.com/digitalbazaar/vc/blob/9975a7b9184bb50b179566517f9d480a5d30170e/lib/index.js#L77-L78
vs.
https://w3c-ccg.github.io/vc-api/#verify-presentation

Not sure the ramifications of changing that at this point...

Reply:
I updated this one, but am holding off on a larger exact match project. We can update these if/when/as we find deviations.